lib/node_modules/@stdlib/math/generics/statistics/incrprod/lib/incrprod.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var frexp = require( '@stdlib/math/base/special/frexp' );
+var ldexp = require( '@stdlib/math/base/special/ldexp' );
+
+/**
+ * Returns an accumulator function which incrementally computes a product.
+ *
+ * @returns {Function} accumulator function
+ *
+ * @example
+ * var accumulator = incrprod();
+ */
+function incrprod() {
+    var product = 1.0;
+    var productFactorized = frexp( product );
+    /**
+     * If provided a value, the accumulator function returns an updated product. If not provided a value, the accumulator function returns the current product.
+     *
+     * @param {number} [x] - new value
+     * @returns {number} product
+     *
+     * @example
+     * var product = accumulator();
+     * // returns 1.0
+     *
+     * product = accumulator( 2.0 );
+     * // returns 2.0
+     *
+     * product = accumulator( -5.0 );
+     * // returns -10.0
+     *
+     * product = accumulator();
+     * // returns -10.0
+     */
+    return function accumulator( x ) {
+        if ( arguments.length === 0 ) {
+            return product;
+        }
+
+        var newFactorized = frexp( x );
+        // Multiply mantissa
+        productFactorized[ 0 ] *= newFactorized[ 0 ];
+        // Add exponents
+        productFactorized[ 1 ] += newFactorized[ 1 ];
+
```

Comment:
That's correct, I figured that out a couple hours after issuing the pull request.

It should be a trivial expansion.